src/app/window-handler.ts

```diff
@@ -241,6 +244,13 @@ export class WindowHandler {
             }
             this.url = this.mainWindow.webContents.getURL();
 
+            logger.info(`window-handler: client switch from config is ${this.config.clientSwitch}`);
+
+            const parsedUrl = parse(this.url);
+            if (this.url.indexOf(`https://${parsedUrl.hostname}/client/index.html`) !== -1) {
```

Comment:
I think we should also add this check `this.url.startsWith('https://corporate.symphony.com')` here as well right?